src/LandingPage/CreateUserModal.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import './LandingPage.css';
+import axios from "axios";
+import { connect } from "react-redux";
+import Modal from 'react-responsive-modal';
+
```

Comment:
Can you collapse this spacing? Usually there should be two lines between imports and the class and then one line between methods.

Reply:
done